server/.env

```diff
@@ -1,2 +1,2 @@
-TEAM_NAME="Shums"
-FLASK_ENV="development"
+TEAM_NAME="flamingo"
+DB_CONNECTION_STRING="postgresql://postgres:password@localhost:5432/devdb"
```

Comment:
what happened is I have my FLASK variables in another file. Since .env is specific to dotenv,
I'm thinking of excluding this file from this branch, so it doesn't create unnecessary dependency on the other PRs. (It's not clear to me if the final .env should contain all our names; also the default .gitignore says to add .env in the ToDo section).  In general, I favor not dictating to users/admin of the app how to set their environment variables. Maybe I'll change my mind later on when I see the wisdom of this.

Reply:
.env is actually specified by pipenv as well. "pipenv run" loads the .env file as well. I think we should keep the .env file in server, and run the server with pipenv run, as that's all set up. 
The final TEAM_NAME should have all our names in it, but I imagine it won't be in the final app -- it's there to help us get familiar with the tech stack, PR, and merging. 
Since we're the admin of the app, and the users of the app don't change the environment variables, we can set it as we please.

Reply:
.env is included as pipenv. We should use "pipenv run", which loads the .env file. Since we're the admins of this app, we should set up the environment variables that suits us best.

Reply:
I agree that we should exclude this from the branch for reasons you mentioned.